src/components/StatesButton/StatesButton.st.css

```
@@ -21,6 +22,10 @@
     display: block;
 }
 
+.root:upgrade::content{
+    line-height: 1;
```

This needs to be added:
```suggestion
    line-height: 1;
    display: inline-block;
```